src/consensus/params.h

```diff
@@ -12,8 +12,20 @@ namespace Consensus {
 /**
  * Parameters that influence chain consensus.
  */
+
+class CDynamicParams
```

Comment:
This class should't be needed: CChainParams::consensus is not const (even though CChainParams::GetConsensus() returns a const reference). See #6382.